### PR TITLE
fixed typos

### PR DESCRIPTION
**Pull request by Ahmad495**

### CODE REVIEW

**Comment by akshay-joshi** [comment]
@Ahmad495 

No need to change the *.po file. Please update the PR and make changes only to the source code files

**Comment by Ahmad495** [comment]
> @Ahmad495
> 
> No need to change the *.po file. Please update the PR and make changes only to the source code files

removed the changes made to .po files.

File: web/pgadmin/authenticate/mfa/views.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def __handle_mfa_validation_request(
     mfa_method: str, user_mfa_auths: dict, form_data: dict
 ) -> None:
     """
-    An internal utlity function to execute mfa.validate(...) method in case, it
+    An internal utility function to execute mfa.validate(...) method in case, it
```
Comment (akshay-joshi) [inline web/pgadmin/authenticate/mfa/views.py:40]: Fixed the PEP8 issue as this line is more than 80 columns.

**Review by akshay-joshi — CHANGES_REQUESTED** [review]

**Comment by akshay-joshi** [comment]
@Ahmad495 

Could you please fix this and send the PR today? I'll generate the .po file and commit it for the translations.

**Comment by Ahmad495** [comment]
> @Ahmad495
> 
> Could you please fix this and update the PR today? I'll generate the .po file and commit it for the translations.

Fixed PEP8 issue.

**Pull request merged**